### DISCUSSION TITLE
Strategies for using firewall (WAF) with Livewire

### DISCUSSION THREAD

**Discussion by twarkie** [livewire/livewire#7084] [1. Help]
I've been using Cloudflare for a long time and I like their **WAF** (Web Application Firewall). It allows me to set up rules to protect me against various attacks - brute forcing for example. 

I haven't found a way to set up a rate limiting WAF rule with Livewire because of the single endpoint Livewire uses (default: `/livewire/update`). I am of course throttling the requests in Laravel but it's not really the same. It would be nice to stop the attempts directly on Cloudflare. 

Just wondering how other people are handling this. Has anyone been able to set up a WAF rule together with Livewire?

**Comment by papa-ree** [comment]
how about bypass POST request method (path livewire/update) in security menu?
i use that rule, just work
